### PR TITLE
Option to hide the issuer and client header titles

### PR DESCRIPTION
**Pull request by dragonfly4**
Hide the issuer and client header row

### CODE REVIEW

File: src/InvoicePrinter.php [hunk 1]
```diff
@@ -201,6 +202,11 @@ public function hide_tofrom()
     {
         $this->display_tofrom = false;
     }
+	
+	public function hide_tofromheaders()
```
Comment (fulopattila122) [inline src/InvoicePrinter.php]: please use camel case method names: `hideToFromHeaders()`

File: src/InvoicePrinter.php [hunk 2]
```diff
@@ -49,6 +49,7 @@ class InvoicePrinter extends FPDF
     public $footernote;
     public $dimensions;
     public $display_tofrom = true;
+    public $display_tofromheaders = true;
```
Comment (fulopattila122) [inline src/InvoicePrinter.php]: please use camel case variable names for new elements. The previous one is kept as-is for compatibility only (and it's quite screwed up :smiley:  )

I suggest:

```php
protected $displayToFromHeaders = true;
```

File: src/InvoicePrinter.php (hunk 1 above)
Comment (dragonfly4) [reply src/InvoicePrinter.php]: DONE

`public function hideToFromHeaders()`

File: src/InvoicePrinter.php (hunk 2 above)
Comment (dragonfly4) [reply src/InvoicePrinter.php]: DONE

`protected $displayToFromHeaders = true;`

**Comment by dragonfly4** [comment]
Update with requested changes has been pushed

**Pull request merged**